src/output/output.hpp

```diff
@@ -40,6 +43,7 @@ class Output {
  private:
   Vtk vtk;          // local instance of Vtk class
   Dump dump;        // local instance of Dump class
+  Xdmf xdmf;        // local instance of Xdmf class
```

Comment:
Same. To be included on demand. Not by default.

Reply:
@glesur @neutrinoceros Can you suggest a way to do this?

Reply:
@glesur I have already put this to be optionally included if the pre-processor flag `USE_HDF5` is defined in `CMakeLists.txt` in the root directory of Idefix.